client/builtin_roles.go

```diff
@@ -0,0 +1,12 @@
+package client
+
+const (
+	Admin    string = "Admin"
+	Deployer string = "Deployer"
+	Planner  string = "Planner"
+	Viewer   string = "Viewer"
```

Comment:
these are common for both organization/project/environment ?

Reply:
not at this stage.
Not all are allowed for each of the types.

There will be a follow-up PR that will leverage these.